libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -364,6 +365,24 @@ int32_t AP_Landing_Deepstall::get_target_airspeed_cm(void) const
     }
 }
 
+bool AP_Landing_Deepstall::send_deepstall_message(mavlink_channel_t chan) const
+{
+//    CHECK_PAYLOAD_SIZE(DEEPSTALL);
```

Comment:
Ah, yes that should be a fixme... CHECK_PAYLOAD_SIZE doesn't work, because it's a macro that can't be included anywhere other then a class that inherits from GCS.... I'm open to suggestions but you can't compile with that macro, and sticking it in the ArduPlane layer is annoying as its a implicit knowledge leak.